home/templates/home/home.html

```diff
@@ -9,7 +9,9 @@
     <div class="hero-content">
         <h1>Welcome to ConnectHer</h1>
         <p><strong>find</strong> or <strong>give</strong> support <br> <strong>learn</strong> or <strong>share</strong></p>
-        <a href="#" class="cta-button">Explore</a>
+        <form action="endometriosis.html">
```

Comment:
This needs to be an <a> element. I created another page for you called learn.html.

To link to this in Django you use this syntax:
 `<a href="{% url 'nameofurl' %}"</a>`

So in this case it would be `<a href="{% url 'learn' %}"</a>`